index.html

```diff
@@ -189,17 +219,23 @@ <h2>Prepare for engaging during the meeting</h2>
           <div class="text">
                Join the following Slack channels:
                <ul>
-               <!-- collab meeting channel -->
+               <!-- hack-sprint cannel -->
                <li>
-                   <a href="https://lsstc.slack.com/messages/C90AMAG80/" title="link to LSSTC Slack">#desc-collab-meeting</a> to hear meeting announcements and to interact with meeting participants.
+                   <a href="https://lsstc.slack.com/messages/C2M8FG5A6"
+                   title="link to LSSTC Slack">#desc-hack-sprint</a>
+                    to hear announcements and to interact with with all the sprinters.
                </li>
-               <!-- hack-sprint -->
+               <!-- tech hotline -->
                <li>
-                  <a href="https://lsstc.slack.com/messages/C2M8FG5A6/" title="link to LSSTC Slack">#desc-hack-sprint</a> to hear Sprints-related announcements and to interact with Sprinters.
+                  <a href="https://app.slack.com/messages/C01EY4AK7UJ"
+                  title="link to LSSTC Slack">#desc-sprint-hotline</a>
+                    to report any technical problems that may arise during the sprint week.
                </li>
-               <!-- hotline -->
+               <!-- science hotline -->
                <li>
-                  <a href="https://app.slack.com/client/T06D204F2/C0176US657B/details" title="link to LSSTC Slack">#desc-collab-meeting-hotline</a> to report any technical problems that may arise during the meeting.
+                  <a href="https://app.slack.com/messages/C01FAP8T1PB"
+                  title="link to LSSTC Slack">#desc-sprint-science-help</a>
```

Comment:
I wonder if this might be more slack channels than we need?  I would imagine, for example, that some questions might more naturally be asked in pre-existing ones, like a CCL question in #desc-ccl and a DC2 question in #desc-dc2-users, and then the few science questions with no obvious home could go in #desc-hack-sprint?  (It might even be easier for people not attending synchronously to engage with questions in those other channels?)

Reply:
so the thinking within the LOC was that #desc-hack-sprint might be too busy with announcements etc. s.t. new members who want to get science help might not want to post their inquiries these. i can ask the LOC re further thoughts on this.